worker/pkg/workflows/datasync/activities/activities_test.go

```diff
@@ -360,7 +360,7 @@ func Test_buildProcessorConfigMutation(t *testing.T) {
 	})
 
 	assert.Nil(t, err)
-	assert.Equal(t, *output.Mutation, "root.id = null\nroot.name = null")
+	assert.Equal(t, *output[0].Mutation, "root.id = null\nroot.name = null")
```

Comment:
Minor thing but you could add a new line above this that asserts the length of the output is 1, since you are expecting in your test that only a mutation config should be added.

```
assert.Len(t, output, 1)
assert.Equal(t, *output[0].Mutation, "root.id = null\nroot.name = null")
```

Then, for any test where you are expecting a mutation _and_ js processor, you can assert that the length should be 2. and so on. Although i dont see a test that handles the case where both are present. Would be great to have one.

Reply:
Updated. Test that handles both is in the benthos-builder_test.go called `Test_ProcessorConfigMutationAndJavascript`